jetty-integrations/jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -332,48 +335,37 @@ public boolean doExists(String id) throws Exception
         Long expiry = (Long)sessionDocument.get(__EXPIRY);
 
         //expired?
-        if (expiry.longValue() > 0 && expiry.longValue() < System.currentTimeMillis())
+        if (expiry > 0 && expiry < System.currentTimeMillis())
```

Comment:
Won't this throw NPE if `expiry` is `null` ?

Reply:
Previous code as well :) so I presume it,s never null but I can add some null check.